PocketKit/Sources/Sync/PocketSource.swift

```diff
@@ -642,3 +642,10 @@ extension PocketSource {
         }
     }
 }
+
+// MARK: - Search term
+extension PocketSource {
+    public func searchSaves(search: String, isPremium: Bool) -> [SavedItem]? {
+        try? space.fetchSavedItems(bySearchTerm: search, userPremium: isPremium)
```

Comment:
what do you think about using `user.status == .premium` as opposed to passing `isPremium` since sync should have a user object?

This way we can simplify the call in the `SearchViewModel` and trust that source will return the right data. Similar to how we rely on the server to provide us the right items via Apollo.

```
let list = self.source.searchSaves(search: term)
```

Reply:
didn't see any modifications related to this, did I miss the conclusion we decided to take?